storage-access.bs

```diff
@@ -310,17 +310,47 @@ The Storage Access API defines a [=policy-controlled feature=] identified by the
 
 <h2 id="privacy">Privacy considerations</h2>
 
-ISSUE: Write this section.
+The Storage Access API is meant able to enable the process of third-party cookie deprecation. Specifically, it enables the authenticated embed use case to continue to work, albeit under further constraints. When evaluating the privacy properties of the Storage Access API it is important to consider the pre-deprecation world of third-party cookies as the baseline. However, we aim to provide strong anti-tracking protections to users, as to not undermine the gains of third-party cookie deprecation.
+
+Importantly, we do not degrade privacy properties when compared to pre-deprecation of third-party cookies. This follows from a lack of platform-specific information used in the spec to prevent stateless tracking and the only state added being a permission scoped to the [=site|sites=] of the embedding and embedded [=Document=].
+
+Our privacy considerations are more challenging where default third-party cookies are already deprecated. The challenge is to decide when and how to permit the Storage Access API to be used to revert a cookie-less (or cookie-partitioned) iframe to a pre-deprecation state, giving it access to its [=unpartitioned data=].
+
+In an ideal case, an iframe would only be able to gain access to its [=unpartitioned data=] if:
```

Comment:
I'm not sure if we should mirror the required steps in the algorithm here, we should try to have a single source of truth and just link to the steps in the determine storage access algorithm.

Reply:
I think keeping abstract points here may be useful because they can guide iteration on the algorithms. I do share your concern about having more than one source of truth.

Reply:
I think we should talk about a nested document here. "iframe" isn't really a term-of-art and also doesn't hold the authority.